torchao/experimental/ops/packed_weights_header.h

```diff
@@ -10,44 +10,47 @@
 #include <cassert>
 namespace torchao::ops {
 
-enum PackedWeightsFormat : unsigned short {
+enum PackedWeightsFormat : int {
```

Comment:
Why this must be a signed type? `unsigned int` sounds like a better idea. Also `enum` inside the namespace is oxymoron, please use `enum class`
```suggestion
enum class PackedWeightsFormat : uint32_t {
```